docs/commands/build-docs.md

```diff
@@ -16,76 +17,80 @@ redocly build-docs <api> -t custom.hbs --templateOptions.metaDescription "Page m
+| --template, -t      | string  | Use custom [Handlebars](https://handlebarsjs.com/) templates to render your OpenAPI description.                                                                                                                                   |
+| --templateOptions   | string  | Add template options you want to pass to your custom Handlebars template. To add options, use dot notation.                                                                                                                        |
+| --theme.openapi     | string  | Customize your output with [Redoc functionality options](https://redocly.com/docs/api-reference-docs/configuration/functionality/) or [Redoc theming options](https://redocly.com/docs/api-reference-docs/configuration/theming/). |
+| --title             | string  | Set the page title.                                                                                                                                                                                                                |
+| --version           | boolean | Show version number.                                                                                                                                                                                                               |
 
 ## Examples
 
-### API examples
+### Specify API
 
-The command accepts an API positional argument as either a file (no configuration file is required) or an alias (requires a [configuration file](#custom-configuration-file)).
+The `build-docs` command behaves differently depending on how you pass the API to it, and whether the [configuration file](#use-custom-configuration-file) exists.
 
-#### API path to file example
+#### Pass API directly
 
 ```bash
 redocly build-docs openapi.yaml
 ```
 
-In this case, the `build-docs` command builds the API at the path provided.
+In this case, the `build-docs` command builds the API description that was passed to the command.
 The configuration file is ignored.
 
-#### API alias example
+#### Pass API alias
 
 Instead of a full path, you can use an API name from the `apis` object of your Redocly configuration file.
-
-```bash Command
-redocly build-docs games@v1
-```
+For example, with a `redocly.yaml` configuration file containing the following entry for `games@v1`:
 
 ```yaml Configuration file
 apis:
   games@v1:
     root: ./openapi/api-description.json
 ```
 
-The `build-docs` command uses any additional configurations provided in the file.
+You can generate a build by giving the API alias name, as shown below:
+
+```bash Command
+redocly build-docs games@v1
+```
+
+In this case, after resolving the path behind the `games@v1` name, `build-docs` generates a build of the `api-description.json` file. For this approach, the Redocly configuration file is mandatory.
+Any additional configurations provided in the file are used by the command.
 
-### Custom configuration file
+### Use custom configuration file
```

Comment:
```suggestion
### Use alternative file path
```

Reply:
Updated per suggestion.

Reply:
Corrected to "filepath" to be consistent with the other command docs and [English convention](https://english.stackexchange.com/questions/428215/filepath-or-file-path).